.github/workflows/pull.yml

```diff
@@ -175,6 +175,32 @@ jobs:
       docker-image: ${{ needs.linux-focal-py3_11-clang10-build.outputs.docker-image }}
       test-matrix: ${{ needs.linux-focal-py3_11-clang10-build.outputs.test-matrix }}
 
+  linux-focal-py3_12-clang10-build:
+    name: linux-focal-py3.12-clang10
+    uses: ./.github/workflows/_linux-build.yml
+    with:
+      build-environment: linux-focal-py3.12-clang10
+      docker-image-name: pytorch-linux-focal-py3.12-clang10
+      test-matrix: |
+        { include: [
+          { config: "default", shard: 1, num_shards: 3, runner: "linux.2xlarge" },
+          { config: "default", shard: 2, num_shards: 3, runner: "linux.2xlarge" },
+          { config: "default", shard: 3, num_shards: 3, runner: "linux.2xlarge" },
+          { config: "crossref", shard: 1, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "crossref", shard: 2, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "dynamo", shard: 1, num_shards: 2, runner: "linux.2xlarge" },
+          { config: "dynamo", shard: 2, num_shards: 2, runner: "linux.2xlarge" },
```

Comment:
Can we really run `dynamo` tests in 3.12?

Reply:
No, I was curious what was going to be the failure and wanted to make sure it failed nicely.